python/pyproject.toml

```diff
@@ -16,15 +16,15 @@
 # under the License.
 
 [build-system]
-requires = ["maturin>=0.15,<0.16"]
+requires = ["maturin>=1.5.1,<1.6.0"]
 build-backend = "maturin"
 
 [project]
 name = "ballista"
 description = "Python client for Apache Arrow Ballista Distributed SQL Query Engine"
 readme = "README.md"
 license = {file = "LICENSE.txt"}
-requires-python = ">=3.6"
+requires-python = ">=3.7"
```

Comment:
This would conclict with #1136 so I will change it really quick and repush my branch.

Reply:
updated #1136 to require python 3.7 or greater

Reply:
Datafusion python has python > 3.7